projects/plugins/jetpack/modules/widget-visibility/widget-conditions/widget-conditions.js

```diff
@@ -1,10 +1,17 @@
 /* global isRtl, widget_conditions_parent_pages, widget_conditions_data, jQuery */
 
 jQuery( function ( $ ) {
-	var widgets_shell = $( 'div#widgets-right' );
+	//  Gutenberg widgets + customizer screen.
+	var widgets_shell = $( '#widgets-editor' );
 
-	if ( ! widgets_shell && ! widgets_shell.length ) {
-		widgets_shell = $( 'form#customize-controls' );
+	if ( 0 === widgets_shell.length ) {
+		// Pre-gutenberg widget screen.
+		widgets_shell = $( 'div#widgets-right' );
```

Comment:
It seems that `$( 'div#widgets-right' )` exists in "Pre-gutenberg widget screen" and in both pre/after versions of customizer (in WP 5.7 and WP 5.8). 

In that case I would just change the comments to reflect that. For example  `$( '#widgets-editor' )` doesn't exist in the customizer.

Let's leave `$( 'form#customize-controls' )` as is for backwards compatibility,

Reply:
Ok, I've made this change if I understood correctly

Reply:
Pretty close, I added my suggestions ;-)

Reply:
Thanks :)